mava/utils/logger.py

```diff
@@ -393,18 +393,19 @@ def write(
 
         metrics = {metric_key: [value]}
 
-        if logging_prefix == "evaluator":
-            step_metrics = {"step_count": timestep, "elapsed_time": current_time - self.start_time}
-            step_metrics.update(metrics)  # type: ignore
+        if logging_prefix == "absolute":
+            self.run_data["absolute_metrics"].update(metrics)
+
+        elif logging_prefix == "evaluator":
+            step_metrics = {
+                "step_count": timestep,
+                "elapsed_time": current_time - self.start_time,
+            } | metrics
```

Comment:
I definitely change this in #994. Can we get that in first? And then make the change on top of that?

Reply:
Scratch that I was thinking of the other logging PR, so never mind this. Will approve :smile: